### PR TITLE
Adding resolutions for security critical packages

### PR DESCRIPTION
**Pull request by georgewrmarshall**
**Description**

`yarn audit` and `yarn improved-yarn-audit` are failing. This PR updates those security critical packages by adding them to resolutions.

Maybe if these are merged we can remove them from the resolutions list:

https://github.com/MetaMask/metamask-mobile/pull/4435
https://github.com/MetaMask/metamask-mobile/pull/4434

**Checklist**

* ~[x] There is a related GitHub issue~ NA
* ~[x] Tests are included if applicable~ NA
* ~[x] Any added code is fully documented~ NA

**Screenshots/Recordings**

### Before 
Screen Shot 2022-06-03 at 11 47 43 AM

### After
Screen Shot 2022-06-03 at 1 20 28 PM

#### Storybook actions still work

https://user-images.githubusercontent.com/8112138/171947733-7216c8e3-1dee-4790-adb1-6c81994daa8d.mov

### CODE REVIEW

File: .iyarc [hunk 1]
```diff
@@ -1,3 +1 @@
 # improved-yarn-audit advisory exclusions
-GHSA-fwr7-v2mv-hh25
-GHSA-gff7-g5r8-mg8m
```
Comment (georgewrmarshall) [inline .iyarc:3]: Fixes 
Screen Shot 2022-06-03 at 1 22 15 PM

File: app/images/static-logos.js [hunk 2]
```diff
@@ -500,4 +544,16 @@ export default {
...
+  'nearpad.svg': require('metamask/node_modules/@metamask/contract-metadata/images/nearpad.svg'),
+  'DLB.svg': require('metamask/node_modules/@metamask/contract-metadata/images/DLB.svg'),
+  'APE.svg': require('metamask/node_modules/@metamask/contract-metadata/images/APE.svg'),
+  'torum.svg': require('metamask/node_modules/@metamask/contract-metadata/images/torum.svg'),
+  'floki.svg': require('metamask/node_modules/@metamask/contract-metadata/images/floki.svg'),
+  'nexum.svg': require('metamask/node_modules/@metamask/contract-metadata/images/nexum.svg'),
+  'mfg.svg': require('metamask/node_modules/@metamask/contract-metadata/images/mfg.svg'),
+  'rETH.svg': require('metamask/node_modules/@metamask/contract-metadata/images/rETH.svg'),
+  'rpl.svg': require('metamask/node_modules/@metamask/contract-metadata/images/rpl.svg'),
 };
```
Comment (georgewrmarshall) [inline app/images/static-logos.js:559]: Not sure what this file does exactly but seems like it was updated as a result of the `"react-native-svg-asset-plugin/sharp": "^0.30.5"` resolution

File: package.json [hunk 3]
```diff
@@ -94,7 +96,12 @@
     "**/plist": "3.0.5",
     "**/minimist": "1.2.6",
     "@metamask/**/cross-fetch": "2.2.6",
-    "react-native-gesture-handler/**/cross-fetch": "3.1.5"
+    "react-native-gesture-handler/**/cross-fetch": "3.1.5",
+    "@storybook/addon-actions/**/prismjs": "^1.27.0",
```
Comment (georgewrmarshall) [inline package.json:100]: https://github.com/advisories/GHSA-3949-f494-cm99

File: package.json [hunk 4]
```diff
@@ -94,7 +96,12 @@
...
+    "@storybook/addon-actions/**/highlight.js": "^10.4.1",
```
Comment (georgewrmarshall) [inline package.json:101]: https://github.com/advisories/GHSA-vfrc-7r7c-w9mx

File: package.json [hunk 5]
```diff
@@ -94,7 +96,12 @@
...
+    "react-native-svg-asset-plugin/sharp": "^0.30.5",
```
Comment (georgewrmarshall) [inline package.json:102]: https://github.com/advisories/GHSA-gp95-ppv5-3jc5

File: package.json [hunk 6]
```diff
@@ -94,7 +96,12 @@
...
+    "detox/**/moment": "^2.29.2",
```
Comment (georgewrmarshall) [inline package.json:103]: https://github.com/advisories/GHSA-8hfj-j24r-96c4

File: package.json [hunk 7]
```diff
@@ -94,7 +96,12 @@
...
+    "oss-attribution-generator/**/debug": "^2.6.9"
```
Comment (georgewrmarshall) [inline package.json:104]: https://github.com/advisories/GHSA-gxpj-cx7g-858c

File: package.json [hunk 8]
```diff
@@ -69,6 +69,8 @@
     "**/nanoid": "^3.1.31",
     "**/json-schema": "^0.4.0",
     "@sentry/react-native/**/xmldom": "^0.6.0",
+    "@sentry/**/simple-plist": "^1.3.1",
```
Comment (georgewrmarshall) [inline package.json:72]: https://github.com/advisories/GHSA-gff7-g5r8-mg8m

File: package.json [hunk 9]
```diff
@@ -69,6 +69,8 @@
...
+    "react-native/**/simple-plist": "^1.3.1",
```
Comment (georgewrmarshall) [inline package.json:73]: https://github.com/advisories/GHSA-gff7-g5r8-mg8m

Comment (georgewrmarshall) [inline package.json:73]: https://github.com/advisories/GHSA-gff7-g5r8-mg8m

**Review by rickycodes — APPROVED** [review]
This all LGTM.

Thanks for taking care of this @georgewrmarshall

**Pull request merged**